src/Microsoft.Framework.Configuration.Binder/ConfigurationBinder.cs

```diff
@@ -25,20 +25,30 @@ public static void Bind(this IConfiguration configuration, object instance)
             }
         }
 
-        public static T Get<T>(this IConfiguration configuration)
+        public static T Get<T>(this IConfiguration configuration, T defaultValue)
```

Comment:
default is a keyword or we could use it.  defaultValue is pretty understood in general, I don't think T/defaultValue is bad at all